packages/ui-scripts/package.json

```diff
@@ -24,7 +24,6 @@
   "license": "MIT",
   "dependencies": {
     "@babel/cli": "^7.8.4",
-    "@codesandbox/common": "^1.0.8",
```

Comment:
This dependency was breaking the build when using Yarn 3.2.0. It was only used for a command like script to create CodeSandbox-es, which we never used, so I daleted it (it was not updated since a while)